sacred/observers/file_storage.py

```diff
@@ -49,11 +49,16 @@ def __init__(self, basedir, resource_dir, source_dir, template,
         self.cout = ""
         self.cout_write_cursor = 0
 
-    def _make_next_dir(self):
+    def _maximum_existing_run_id(self):
         dir_nrs = [int(d) for d in os.listdir(self.basedir)
                    if os.path.isdir(os.path.join(self.basedir, d)) and
                    d.isdigit()]
-        _id = max(dir_nrs + [0]) + 1
+        if len(dir_nrs) == 0:
```

Comment:
```suggestion
        if not dir_nrs:
```
would be more pythonic, or probably 
```python
if dir_nrs:
   return max(dir_nrs)
else:
   return 0
```

Reply:
Thanks, adjusted it.